plugins/workspace-minimap/test/coordinate_conversion.mocha.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+const chai = require('chai');
+const assert = chai.assert;
+const {Minimap} = require('../src/minimap');
+
+
+suite('Interactions Suite (Shape, Size, Location)', function() {
+  suite('Square, Medium, Center', function() {
+    setup(function() {
+      this.primaryMetrics = {contentHeight: 1000, contentWidth: 1000,
+        contentTop: -500, contentLeft: -500, viewWidth: 500, viewHeight: 500};
+      this.minimapMetrics = {svgWidth: 500, svgHeight: 500,
+        contentHeight: 500, contentWidth: 500};
+    });
+
+    test('Top left click', function() {
+      const click = {x: 0, y: 0};
+      const converted = Minimap.minimapToPrimaryCoords(
+          this.primaryMetrics, this.minimapMetrics,
+          click.x, click.y);
+
+      assert.strictEqual(
+          converted.toString(),
+          [750, 750].toString(),
+          'Incorrect top left click');
```

Comment:
I believe when a test fails, the message is something like "Expected <message>" So this would read "Expected 'Incorret top left click'" which is probably not what you want :P but you can call `assert.fail()` here  to force a failure and double check me!

Reply:
The following is a sample output if one of the tests fail:


 ```
 1) Interactions Suite (Shape, Size, Location)
       Square, Medium, Center
         Top left click:

Incorrect top left click
      + expected - actual

       [
      +  850
         750
      -  750
       ]
```

Reply:
Sweet sounds good then!